### PR TITLE
Big Query downloader tool cleanup + comments

### PR DESCRIPTION
**Pull request by Miking98**
Added some comments for the Bigquery downloader tool to explain how to use it

### CODE REVIEW

**Comment by Miking98** [comment]
Functionality should be identical, just added some comments to clear up some initial confusion I had running through this

File: tools/stanford/download_bigquery.py [hunk 1]
```diff
@@ -7,81 +24,124 @@
...
-    storage_client = storage.Client(project=args.project)
-
-    dataset_id = f"{args.project}.{args.dataset}"
-
-    bucket_name = f"{args.project}-temp-extract"
+    # Connect to our BigQuery project
+    client = bigquery.Client(project=args.gcp_project_name)
+    storage_client = storage.Client(project=args.gcp_project_name)
 
-    os.mkdir(dataset_id)
```
Comment (Lalaland) [inline tools/stanford/download_bigquery.py:36]: Why did you remove this?

**Review by Lalaland — APPROVED** [review]
Looks good to me with two minor comments / issues!

File: tools/stanford/download_bigquery.py [hunk 2]
```diff
@@ -1,4 +1,21 @@
-"""A tool for downloading datasets from BigQuery"""
+"""
+A tool for downloading datasets from BigQuery.
+
+IMPORTANT NOTE: This only works for Python <=3.9, which is NOT
```
[the remark this answers is not in the document]
Comment (Miking98) [reply tools/stanford/download_bigquery.py:4]: I'm pretty sure google cloud does not work with Python 3.10 -- I kept getting an error trying to install it (same error as documented here: https://issuetracker.google.com/issues/205238176?pli=1

File: tools/stanford/download_bigquery.py [hunk 3]
```diff
@@ -7,81 +24,124 @@
...
+        print(f"Download Finished | table = {table_id}")
         sem.release()
 
-    print("Tables contained in '{}':".format(dataset_id))
+    n_tables: int = 0
     for table in tables:
-        if table.table_id in ("note", "note_nlp", "observation"):
+        # Get the full name of the table
+        table_name: str = f"{table.project}.{table.dataset_id}.{table.table_id}"
+        if table.table_id in ["observation"]:
```
[the remark this answers is not in the document]
Comment (Miking98) [reply tools/stanford/download_bigquery.py:125]: Good call yep!

**Review by Lalaland — APPROVED** [review]
